backend/app/services/execution/dispatcher.py

```
@@ -0,0 +1,734 @@
+        db = SessionLocal()
+        try:
+            subtask = db.query(Subtask).filter(Subtask.id == subtask_id).first()
+            if subtask:
+                subtask.executor_name = f"device-{device_id}"
+                subtask.executor_namespace = f"user-{user_id}" if user_id else None
+                db.commit()
+                logger.info(
+                    f"[ExecutionDispatcher] Set executor on subtask {subtask_id}: "
+                    f"executor_name=device-{device_id}"
+                )
+        except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)